### PR TITLE
Bump nyxx + add `acronym` to guild

### LINKED ISSUE #6: Support nyxx 4.0.0

### PR DESCRIPTION
**Pull request by Rapougnac**
# Description
Resolves #6 

## Connected issues & potential other potential problems
-/-

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] Ran `dart analyze` or `make analyze` and fixed all issues
- [x] Ran `dart format --set-exit-if-changed -l 160 ./lib` or `make format` and fixed all issues
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ x I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my changes haven't lowered code coverage

### CODE REVIEW

**Comment by l7ssha** [comment]
Can you add changelog? since its gonna be single change in this version

**Comment by Rapougnac** [comment]
Done

**Comment by l7ssha** [comment]
You gotta also change version in pubspec

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 3.1.0
+__15.11.2022__
+- Bump nyxx to `4.2.0`
```
Comment (l7ssha) [inline CHANGELOG.md:3]:
```suggestion

- Bump nyxx to `4.2.0`
```

**Review by l7ssha — APPROVED** [review]

**Pull request merged**